modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -164,17 +177,13 @@ public int getPort() {
 
     @Override
     protected void containerIsStarted(InspectContainerResponse containerInfo) {
-        if (recordingMode != VncRecordingMode.SKIP) {
+        if (vncRecordingContainer != null) {
             LOGGER.debug("Starting VNC recording");
-
-            VncRecordingSidekickContainer recordingSidekickContainer = new VncRecordingSidekickContainer<>(this);
-
-            recordingSidekickContainer.start();
-            currentVncRecordings.add(recordingSidekickContainer);
+            vncRecordingContainer.start();
         }
 
         driver = Unreliables.retryUntilSuccess(30, TimeUnit.SECONDS,
-                Timeouts.getWithTimeout(10, TimeUnit.SECONDS,
+                Timeouts.getWithTimeout(1, TimeUnit.SECONDS,
```

Comment:
Not directly related to this PR, but sometimes it fails to init, and this code waits for 10s while 1s is enough

Reply:
👍